components/Workout/Workout.js

```diff
@@ -155,25 +161,49 @@ const ExerciseList = styled.ul`
   padding: 0;
 `;
 
+const ExerciseItemHeadline = styled.li`
+  border-bottom: 3px solid #888;
+  padding: 0.5rem 0;
+  text-align: left;
+  font-size: 1rem;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+`;
+
 const ExerciseItem = styled.li`
   border-bottom: 1px solid #ddd;
   padding: 0.5rem 0;
+  text-align: left;
+  font-size: 0.9rem;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
 `;
 
-const ExerciseName = styled.h3`
+const ExerciseName = styled.span`
+  _flex-grow: 1;
```

Comment:
If you don't need a property, I would just remove it.

Reply:
Just removed! :)